packages/loopring_v3/contracts/core/impl/libtransactions/AccountUpdateTransaction.sol

```diff
@@ -43,7 +45,7 @@ library AccountUpdateTransaction
     struct AccountUpdateAuxiliaryData
     {
         bytes  signature;
-        uint96 maxFee;
+        uint16 feeDiscount;
```

Comment:
I'm not sure if we should move `signature` to the end of the struct, will that reduce gas? The same question also apply to the other AuxiliaryData definition.